docs/recipes/response-patching.mdx

```diff
@@ -18,10 +18,11 @@ const worker = setupWorker(
   rest.get('https://api.github.com/users/:username', async (req, res, ctx) => {
     // Perform an original request to the intercepted request URL
     const originalResponse = await ctx.fetch(req)
+    const originalResponseData = await originalResponse.json()
```

Comment:
That’s a great catch. Could you please confirm this suggestion works?

Asking because MSW used to expose already awaited body stream, but I recall that being changed recently.

Reply:
hey, yeah I can confirm the old code wouldn't work and new code does now work 👍 

I've used as a pass-thru for my local dev environment